packages/contracts-bedrock/contracts/test/L2OutputOracle.t.sol

```diff
@@ -411,6 +414,20 @@ contract L2OutputOracleTest is L2OutputOracle_Initializer {
         vm.expectRevert("L2OutputOracle: cannot delete outputs after the latest output index");
         oracle.deleteL2Outputs(latestOutputIndex - 2);
     }
+
+    function test_deleteL2Outputs_finalized_reverts() external {
+        test_proposeL2Output_proposeAnotherOutput_succeeds();
+
+        // Warp past the finalization period + 1 second
+        vm.warp(block.timestamp + 7 days + 1);
```

Comment:
Can we decouple hardcoded params by calling the finalization period getter here?

Reply:
Yeah- will do after retro :100: